includes/models/model.llms.student.php

```diff
@@ -640,7 +641,9 @@ public function get_enrollment_status( $product_id, $use_cache = true ) {
 			// Get the most recent recorded status.
 			$status = $wpdb->get_var(
 				$wpdb->prepare(
-					"SELECT meta_value FROM {$wpdb->prefix}lifterlms_user_postmeta WHERE meta_key = '_status' AND user_id = %d AND post_id = %d ORDER BY updated_date DESC LIMIT 1",
+					"SELECT meta_value FROM {$wpdb->prefix}lifterlms_user_postmeta " .
+					"WHERE meta_key = '_status' AND user_id = %d AND post_id = %d " .
+					"ORDER BY updated_date DESC, meta_id DESC LIMIT 1",
```

Comment:
concatenation here seems unnecessary and newlines are alright in a query

```suggestion
					"SELECT meta_value FROM {$wpdb->prefix}lifterlms_user_postmeta
					 WHERE meta_key = '_status' AND user_id = %d AND post_id = %d
					 ORDER BY updated_date DESC, meta_id DESC LIMIT 1;",
```

Reply:
Looks good to me.